test/Tests/src/Runtime/Asserts_Spec.enso

```diff
@@ -24,4 +25,11 @@ spec = Test.group "Asserts" pending=on_ci <|
         Runtime.assert <|
```

Comment:
Btw. this test above `should be able to take lambdas as expressions`

where is the lambda here? it's a block, but not a lambda function taking arguments. Shall we maybe rename the test case?

Reply:
Right, it should not take lambda as an expression because `Runtime.assert` treats the `action` parameter as suspended. Renamed in 70b764f10d0dbd1d7edc43832c2a8bbec0e26e64